### ISSUE TITLE
Lab 1 - Negative answers and R^2

### ISSUE THREAD

**Issue by JAHirsh** [Watts-College/cpp-523-spr-2022#1]
I have read the Lewis-Beck chapter several times and I am just completely and utterly lost.
In Lab 1, I understand why I have negative numbers based on the calculations, but the numbers don't match the regression line shown. As Y increases, my prediction of y decreases. Also, I don't understand how to calculate R^2. If anyone is able to give me some insight, I would be so grateful. 

[Lab-01-Hirsh.docx](https://github.com/Watts-College/cpp-523-spr-2022/files/8732277/Lab-01-Hirsh.docx)

**Comment by JasonSills** [comment]
@JAHirsh 

Focus on how you are calculating Yhat . You have derived B0 and B1 in #3 and #1, respectively. The equation you now want to think about is: Yhat = B0 + B1X + e. In #3 you have calculated B0 as -0.6, but in subsequent questions you are not plugging this in as B0. It looks like you are continuing to try to solve for B0, but you need to now plug in the values for B0 and B1 into the equation and solve for Yhat. For example, in #4 you should try plugging in your calculated B0 and your calculated B1 and 14 for X. This would looks something like, Yhat = B0 + B1(14). What is Yhat if you plug in your values?

**Comment by JAHirsh** [comment]
@JasonSills 
Thank you so much for your help. I watched the recording of yesterday's lab and it was also super helpful, especially for understanding 

Would the equation go like this?
Yhat = -0.6 + 1.1(14) + e
Yhat = 14.8 + e

**Comment by JasonSills** [comment]
@JAHirsh,

Yes, you've got it. For the answer you can drop the e and just report the 14.8. The e is here to show that not all of the variance is captured with B0 and B1. There is a lot more going on in Yhat than what is represented and accounted for in B0 and B1 and all of that is in e.
